packages/application/src/services.tsx

```diff
@@ -20,9 +20,9 @@ import * as React from 'react';
 import { RequestHandler } from './requests';
 
 /**
- * A utilities class for various elyra services.
+ * A utility class for handling elyra metadata calls.
  */
-export class FrontendServices {
+export class MetadataService {
```

Comment:
This should be a facade for multiple common things available from the Application package and usable for multiple Elyra packages, so metadata service is not a good name. I rather stay with FrontendServices if we are not going to use Application

Reply:
Ok, I'll change it back to `FrontendServices` then, though if you prefer `ApplicationServices` I believe I would be ok with that. I just want to the classname to be clear it's a "services" class